### PR TITLE
*: fix crud case.

### PR DESCRIPTION
**Pull request by disksing**
When tidb is killed at the same time the crud test removes a user, stability_tester may get an error even if the user is successfully deleted. The fix is to check the notExist errors.

### CODE REVIEW

**Review by xinchi131 — COMMENTED** [review]
LGTM

File: stability-tester/suite/crud.go [hunk 1]
```diff
@@ -68,44 +70,58 @@ func (c *CRUDCase) Execute(db *sql.DB, index int) error {
 	}()
 
 	// Add new users.
-	for i := 0; i < c.cfg.UpdateUsers && c.userIDs.len() < c.cfg.UserCount+c.cfg.UpdateUsers; i++ {
-		id := c.userIDs.allocID()
-		if err := c.createUser(db, id); err != nil {
-			return errors.Trace(err)
+	if c.userIDs.len() <= c.cfg.UserCount {
```
Comment (AndreMouche) [inline stability-tester/suite/crud.go]: It seems when we meet condition `=`,   then there comes ` c.userIDs.len() = c.cfg.UserCount+c.cfg.UpdateUsers`,  that's not conform to the old logic ` c.userIDs.len() < c.cfg.UserCount+c.cfg.UpdateUsers`?

File: stability-tester/suite/crud.go [hunk 2]
```diff
@@ -68,44 +70,58 @@ func (c *CRUDCase) Execute(db *sql.DB, index int) error {
...
-		newUsers = append(newUsers, id)
 	}
 
 	// Delete random users.
-	for i := 0; i < c.cfg.UpdateUsers && c.userIDs.len() > c.cfg.UserCount; i++ {
-		id := c.userIDs.randomID()
-		if err := c.checkUserPostCount(db, id); err != nil {
-			return errors.Trace(err)
+	if c.userIDs.len() > c.cfg.UserCount {
+		for i := 0; i < c.cfg.UpdateUsers; i++ {
```
Comment (AndreMouche) [inline stability-tester/suite/crud.go]: Should  the logic keep consistent with the old ? It seems the condition has been changed.

File: stability-tester/suite/crud.go [hunk 3]
```diff
@@ -68,44 +70,58 @@ func (c *CRUDCase) Execute(db *sql.DB, index int) error {
...
-		deleteUsers = append(deleteUsers, id)
 	}
 
 	// Add new posts.
-	for i := 0; i < c.cfg.UpdatePosts && c.postIDs.len() < c.cfg.PostCount+c.cfg.UpdatePosts; i++ {
-		id := c.postIDs.allocID()
-		author := c.userIDs.randomID()
-		if err := c.addPost(db, id, author); err != nil {
-			return errors.Trace(err)
+	if c.postIDs.len() <= c.cfg.PostCount {
```
Comment (AndreMouche) [inline stability-tester/suite/crud.go]: ditto

**Comment by disksing** [comment]
PTAL @AndreMouche

**Review by AndreMouche — APPROVED** [review]
LGTM

**Pull request merged**